examples/README

```diff
@@ -1,5 +1,29 @@
-Sample of OpSim Psf cutout scripts.
+shear_test_experiments
+July 28, 2015
 
-makepsfs is a shell script which calls the necessary python scripts in the bin directory.
+1.  Sample of OpSim Psf cutout scripts.  
 
-pars and image.fits.gz are sample OpSim input catalog and output image.
+The files pars and image.fits.gz in sample_psfs are an OpSim input catalog and output image.
+makelibrary can be used to create the psf library and index files needed for all of the
+following scripts
+
+2.  Sample Great3, GalSim, measurement, and analysis runs
+
+The file run_params is a sample file which drives all of the scripts.
+
+A subdirectory called base is created and run_params is copied to it.
+
+The runbase scripts can be used to call runGreat, runGal, runMeas, and runAnal,
+or these scripts can be run separately.  
+
+    -- runGreat sets up the GalSim output directory, soft links the required psf libraries,
+       and creates the yaml files for galsim
+    -- runGal runs GalSim on the created yaml files.
+    -- runMeas runs the selected measurement algorithm.  Note that both processShearTest.py
+       and run_params must be set up with the correct algorithm setup and field names.
+    -- runAnal runs statistics on the individual measurement catalogs, and stores a 
+       statistical summary of each run in anal.cat.fits.  This BaseCatalog is cumulative.
```

Comment:
A bit of consistency in the naming conventions for these would be nice.  Probably best to use camelCase for all of them, and not have the parameter file start with `run` since it isn't a script.